deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -546,7 +546,7 @@ spec:
                 - --enable-leader-election
                 - --health-probe-bind-address=:8081
                 command:
-                - ocs-operator
+                - entrypoint
```

Comment:
Don't you need to pass the args as well?

Reply:
args are already added [here](https://github.com/red-hat-storage/ocs-operator/pull/2745/files/242b3c7f2cc8bad2fba15e5ad15fb990186391c6#diff-ee1619a1b7c2a5d91947bd568b782ef4c23f550b587b6e3c85a958b204367146R545-R548):